Automattic/woocommerce-payments
# Tokenized PRBs on shortcode cart and checkout

Fixes #8528

#### Changes proposed in this Pull Request

This implements the tokenized cart PRB on shortcode cart and checkout pages.
This is behind the `_wcpay_feature_tokenized_cart_prb` feature flag. Enable it in your `wp_options`.









#### Testing instructions

* Enable the "Enable Cart-Token implementation for PRBs" flag in your dev tools (you'll need to update them)
* Ensure you have Google Pay/Apple Pay enabled in the merchant's settings
* Go to a product page on your site
* You should be able to use the PRB button to get shipping data & check out on simple products & variable products
* You should be able to add a product to the cart and complete checkout from the shortcode cart and checkout pages

Borrowed from the [initial PDP implementation](https://github.com/Automattic/woocommerce-payments/pull/8644) 
* There shouldn't be a difference in functionality with the older implementation (when the "Enable Cart-Token implementation for PRBs" functionality is disabled) besides what's written below - if you do notice something weird, please let me know!
  * The implementation allows you to check out with the US shipping address, but for some reason the UK and CA addresses aren't working correctly (they fail validation in the backend, despite the values being correct)
  * The itemized amount (tax, shipping, etc.) is _sometimes_ not displayed





-------------------


**Post merge**
